heliopy/data/helper.py

```diff
@@ -138,6 +138,9 @@ def cdf_dict(unit_string):
     ionic_charge = u.def_unit('Charged State', 1.6021766*(10**-19) * u.C)
 
     units = OrderedDict([('ratio', u.dimensionless_unscaled),
+                        ('(years)', u.yr),
+                        ('(days)', u.d),
+                        ('df', u.dimensionless_unscaled),
```

Comment:
I think this might have a complicated unit, which data import method does it come from?